src/App.tsx

```diff
@@ -49,7 +50,26 @@ export default function App() {
       console.error(message);
     }
   });
-
+  if(captchaSuccess){
```

Comment:
Nice 👍 this is perfect !  For what we need, this is more than enough.  

The more 'scalable' option as I'm sure you've already thought would be to use a routing system and make them separate pages with different urls.  There are tradeoffs to both, but ultimately a separate screen with routing is a bit more polished.

Reply:
I think you are right. Lets save that for a refactor